plugins/rbac/src/__fixtures__/mockConditionRules.ts

```diff
@@ -91,7 +91,8 @@ export const mockConditionRules = [
           properties: {
             kinds: {
               type: 'array',
-              items: { type: 'string' },
+              items: { type: 'string' }, // TODO: should be enum?
```

Comment:
@ciiay this actually depends on the plugin devs/maintainer as what condition rules they add to their plugins resource-types.

Reply:
Will remove the comment.